src/Application.php

```diff
@@ -325,10 +325,17 @@ public function initProviders()
         // Make sure we keep our current locale..
         $currentlocale = $this['locale'];
 
-        // Setup Swiftmailer, with optional SMTP settings. If no settings are provided in config.yml, mail() is used.
+        // Setup Swiftmailer, with the selected Mail Transport options: smtp or `mail()`.
         $this->register(new Silex\Provider\SwiftmailerServiceProvider());
-        if ($this['config']->get('general/mailoptions')) {
+        if ($this['config']->get('general/mailoptions/transport') == 'mail') {
+            // Use the 'mail' transport.
+            $this['swiftmailer.transport'] = \Swift_MailTransport::newInstance();
```

Comment:
Unless you *want* to use the PHP `mail()`er and you're sending multiple emails.

**Note:** Why someone would *chose* to use the default mailer is beyond me, but still :grinning: